cmd/ak/cmd/sessions/delete.go

```diff
@@ -0,0 +1,58 @@
+package sessions
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete [session ID] [--fail]",
+	Short:   "Delete non-running session",
+	Aliases: []string{"d"},
+	Args:    cobra.MaximumNArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			id, err := latestSessionID()
+			if err != nil {
+				return err
+			}
+			args = append(args, id)
+		}
+
+		r := resolver.Resolver{Client: common.Client()}
+		s, id, err := r.SessionID(args[0])
+		if err != nil {
+			return err
+		}
+
+		if s != nil {
```

Comment:
Idiomatic Go style is to minimize indentation, so instead of this if/else block, do:

```
if s == nil {
    return ...err...
}

ctx, cancel := ...

Reply:
My intention was to save the error in order to print at the end if --fail option is specified. That's why there is if/else, because there is no `if` that will lead to immediate return

Reply:
rewritten